package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "clean": "git clean -fdX .",
     "build": "rollup -c && tsc -b",
+    "predev": "npm install --prefix ./fixtures/gists-app/",
     "dev": "yarn build && yarn --cwd ./fixtures/gists-app/ dev",
```

Comment:
@chaance `predev` and `dev` scripts still reference the `gists` app. I think we should just remove these two scripts.